### PR TITLE
Panel story

### PR DESCRIPTION
**Pull request by williamdingwall**
adding a panel story and panelImage proptype

### CODE REVIEW

**Comment by williamdingwall** [comment]
@dmccraw @amy-r I'm curious on your thoughts with this approach. Wanted to keep 'panelImage' down to a single prop so set it up to take an object containing a an image src and placeholder source.

File: src/js/components/Panel/Panel.stories.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            on to star in comedy films. He also co-directed Quick Change (1990). Murray garnered additional critical acclaim later 
+            in his career, starring in Lost in Translation (2003), which earned him a Golden Globe and a BAFTA Award for Best Actor, 
+            as well as an Academy Award nomination for Best Actor. He also received Golden Globe nominations for his roles in 
+            Ghostbusters, Rushmore (1998), Hyde Park on Hudson (2012), St. Vincent (2014), and the HBO miniseries Olive Kitteridge 
+            (2014), for which he later won his second Primetime Emmy Award for Outstanding Supporting Actor in a Limited Series or a Movie.
+            </p>
+          </PanelCell>
+          <PanelCell mods='Grid-cell'>
+              <img src='https://www.fillmurray.com/640/360' />
+          </PanelCell>
```
Comment (dmccraw) [inline src/js/components/Panel/Panel.stories.js]: is there a reason you aren't using panelImage here?

File: src/js/components/Panel/Panel.js [hunk 2]
```diff
@@ -39,6 +44,7 @@ Panel.propTypes = {
   children: PropTypes.node.isRequired,
   isStriped: PropTypes.bool,
   maxSize: PropTypes.string,
+  panelImage: PropTypes.object,
```
Comment (dmccraw) [inline src/js/components/Panel/Panel.js]: I wonder if it's worth calling out what keys the panelImage object has.
`
PropTypes.shape({Source, PlaceHolder})
`
?

File: src/js/components/Panel/Panel.stories.js (hunk 1 above)
Comment (williamdingwall) [reply src/js/components/Panel/Panel.stories.js]: Yeah an image in a panel is different from a "panel image". I updated it to headerImage in the next commit to make it a little clearer.

Screen Shot 2019-06-05 at 1 46 11 PM
Screen Shot 2019-06-05 at 1 46 18 PM

On it's just content, the other the image become more a part of the panel

File: src/js/components/Panel/Panel.js (hunk 2 above)
Comment (williamdingwall) [reply src/js/components/Panel/Panel.js]: Oh yeah I like that, that's much clearer

File: src/js/components/Panel/Panel.stories.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      <PanelFooter>
+        <span className='Button Button--negative'>Cancel Button Placeholder</span>
+        <span className='Button Button--primary u-spaceLeftSm'>Confirm Button Placeholder</span>
+      </PanelFooter>
+  </Panel>
+))
+
+stories.add('Panel Image', (() =>
+<Panel 
+  panelImage={ select('Select Image', imageOptions, noImage) }
```
[the remark this answers is not in the document]
Comment (williamdingwall) [reply src/js/components/Panel/Panel.stories.js]: updated to use shape so there's more of an indication of what's expected

**Pull request merged**